usage/test-framework/src/main/java/org/apache/brooklyn/test/framework/SimpleShellCommandTest.java

```diff
@@ -87,21 +90,24 @@
      * If not explicitly configured, the default assertion is a non-zero exit code.
      */
     @SetFromFlag("assertStatus")
-    ConfigKey<Map> ASSERT_STATUS = ConfigKeys.newConfigKey(Map.class, "assert.status",
-            "Assertions on command exit code", Maps.newLinkedHashMap());
+    ConfigKey<List<Map<String, Object>>> ASSERT_STATUS = ConfigKeys.newConfigKey(
```

Comment:
do the old tests still work?  and do we want to be this strict or make best effort to understand -- e.g. if this is `ConfigKey<Object>` we can have a method e.g. `getAssertions(ASSERT_STATUS)` which retrieves it and creates a list of maps, but which would be more forgiving, e.g. if it's a single map we could wrap it in a list.

this is new so i don't mind either way.  i prefer making best-efforts to parse people's intentions and warning but still working if they're wrong (especially if we're changing a syntax which people are already using, as we are here).  however other people in the community prefer fail fast even on user input so either is fine...